app/src/main/java/com/simple/checkstyledemo/MainActivity.java

```diff
@@ -1,13 +1,22 @@
 package com.simple.checkstyledemo;
 
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.app.Application;
+import android.app.*;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        int i = 3;
+        test(i);
+    }
+
+    private void test(int a) {
+        if (a > 3) return;
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.blocks.NeedBracesCheck
**Severity**: ERROR
**File**: app/src/main/java/com/simple/checkstyledemo/MainActivity.java L19

&apos;if&apos; construct must use &apos;{}&apos;s.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*